help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -1620,6 +1620,43 @@ LogixNG: IQ:AUTO:0001
 
       <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
 
+      <h4 id="execute-action">Execute action<span class="since">since 5.7.8</span></h4>
+
+      <p><strong>Execute action</strong> is used to execute a different part of the current
+      ConditionalNG.  This can be useful if you need to execute a part of the ConditionalNG at
+      several different places, for example if you have several <strong>Listen On Beans - Local
+      variable</strong> in the same ConditionalNG.  This is similar to calling a sub-routine.</p>
+
+      <p>A prerequisite is that the <a href="../LogixNG.shtml#show-names">Show system names and user
```

Comment:
No. If the action or expression has a user name, that name will be shown even if this option is disabled.

Reply:
Note the menu item "Change user name" when you right-click on an action or expression.